applications/StructuralMechanicsApplication/custom_constitutive/linear_plane_strain.cpp

```diff
@@ -98,12 +98,43 @@ void  LinearPlaneStrain::CalculateMaterialResponsePK2 (Parameters& rValues)
     }
 }
 
-//note that since we are in the hypothesis of small strains we can use the same function for everything
+//NOTE: Note that since we are in the hypothesis of small strains we can use the same function for everything
+
 void LinearPlaneStrain::CalculateMaterialResponseKirchhoff (Parameters& rValues)
 {
     CalculateMaterialResponsePK2(rValues);
 }
 
+void LinearPlaneStrain::CalculateMaterialResponsePK1 (Parameters& rValues)
+{
+    CalculateMaterialResponsePK2(rValues);
+}
+
+void LinearPlaneStrain::CalculateMaterialResponseCauchy (Parameters& rValues)
+{
+    CalculateMaterialResponsePK2(rValues);
+}
+
+void LinearPlaneStrain::FinalizeMaterialResponsePK2 (Parameters& rValues)
+{
+    // TODO: Add if necessary
```

Comment:
in the base class of the constitutive law there should be a bunch of helper functions for doing pull_back and push_forward operations.

it should be said that it does not really make sense in this context since the CL is only valid in small deformations...